src/index.html

```diff
@@ -2,16 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     onsubmit="onSubmit(Submit)"
+     method="post">
+      <fieldset class="form-field1">
+        <legend>Personal information</legend>
+        <label class="formlabel1" for="surname">
+          Surname:
+          <input id="surname" type="surname">
+        </label>
+        <label class="label" for="name">
+          Name:
+          <input id="name" type="text">
+        </label>
+        <label class="label"  for="number">
+          How old are You ?
+          <input class="form-number"
+           type="number"
+           id="number"
+           value="12"
+           min="0"
+           max="100"
+          >
```

Comment:
```suggestion
          <input class="form-number"
            type="number"
            id="number"
            value="12"
            min="0"
            max="100"
          >
```
відступ від артибута до відкриваючої і закриваючої скобки має бути не один пробіл, а один тег

Reply:
*один tab)